### PR TITLE
[language/kdb+] Minor spelling corrections and subedits

### PR DESCRIPTION
**Pull request by 5jt**
- [x ] Pull request title is prepended with `[language/lang-code]`
- [x ] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x ] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [ ] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [ ] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: kdb+.html.markdown [hunk 1]
```diff
@@ -6,17 +6,17 @@ contributors:
...
+very terse and a little strange-looking for anyone from a "C heritage" language
+background. Its expressiveness and vector-oriented nature suit it
 to performing complex calculations on large amounts of data (while also
 encouraging some amount of [code
 golf](https://en.wikipedia.org/wiki/Code_golf)). The fundamental structure in
 the language is not the object but instead the list, and tables are built as
-collections of lists. This means - unlike most traditional RDBMS systems -
-tables are column oriented.  The language has both an in-memory and on-disk
-database built in, giving a large amount of flexibility. kdb+ is most widely
+collections of lists. This means &ndash; unlike most traditional RDBMS systems &ndash;
```
Comment (samcv) [inline kdb+.html.markdown]: Why did you change them to HTML entities instead of using the actual character? If you're changing it from a normal '-' symbol another please use the actual character not the HTML entity. Also please use the mdash `—` to separate a section of text — ndashes is for ranges of things not for a break in the sentence.

**Comment by vendethiel** [comment]
Closing this for now, please reopen if you get back to it.

File: kdb+.html.markdown (hunk 1 above)
Comment (StephenTaylor-Kx) [inline kdb+.html.markdown]: In the past I’ve found browser support patchy for the ndash and mdash characters – perhaps that’s all behind us now. I’m certainly using them ‘raw’ on the source for code.kx.com, but then I get to preview the output. 

You’re right about the use of the emdash as a separator – in American typography. But it’s normally used—like this—without spaces. Europeans tend to use the endash with spaces. The latter looks better to me when the line breaks on it, but that’s a strictly aesthetic call – whatever you judge best!
